src/pseudopeople/noise_functions.py

```diff
@@ -148,21 +148,51 @@ def miswrite_ages(
 
 
 def miswrite_numerics(
-    form_data: pd.DataFrame,
-    configuration: float,
+    column: pd.Series,
+    configuration: ConfigTree,
     randomness_stream: RandomnessStream,
     additional_key: Any,
-) -> pd.DataFrame:
+) -> pd.Series:
     """
+    Function that noises numeric characters in a series.
 
-    :param form_data:
-    :param configuration:
-    :param randomness_stream:
+    :param column: A pd.Series
+    :param configuration: ConfigTree object containing noise level
+    :param randomness_stream: RandomnessStream for CRN framework.
     :param additional_key: Key for RandomnessStream
-    :return:
+
+    returns: pd.Series with some numeric values experiencing noise.
     """
-    # todo actually duplicate rows
-    return form_data
+
+    token_noise_level = configuration.token_noise_level
+    rng = np.random.default_rng(randomness_stream.seed)
+    column = column.astype(str)
+    longest_str = column.str.len().max()
+    same_len_col = column.str.pad(longest_str, side="right")
+    series_of_lists = same_len_col.str.split("")
+    str_df = pd.DataFrame.from_dict(
+        dict(zip(series_of_lists.index, series_of_lists.values))
+    ).T
+    str_df = str_df.drop(columns=[0, len(str_df.columns) - 1])
+    str_df.columns = list(range(len(str_df.columns)))
+    is_number = str_df.copy()
+    for col in str_df.columns:
+        is_number[col] = is_number[col].str.isdigit()
```

Comment:
We talked about trying to find a better way to turn a Series of strings into a DataFrame of characters. I have a better strategy here:
```
is_number = pd.concat(
    [same_len_col.str[i].str.isdigit() for i in range(longest_str)], axis=1
)
```

Reply:
I had forgot about this but this gets rid of a lot of unnecessary code.